R/dominantTaxa.R

```diff
@@ -0,0 +1,210 @@
+#' Dominant Taxa
+#'
+#' This function returns the most dominant taxa.
```

Comment:
A more verbose description might be beneficial. Esspecially since you have to different return types

Reply:
Changed it to 

'#' These functions return information about the most dominant taxa in a
'#' \code{\link{SummarizedExperiment-class}} object.

Reply:
"Additionally, the information can be directly stored in the \code{colData}"?

Maybe a details section as well, in which you describe the underlying calls regarding data agglomeration